dashboard/lib/policies/script_activity.rb

```diff
@@ -0,0 +1,25 @@
+class Policies::ScriptActivity
```

Comment:
I see great value in abstracting things into Policy and/or Query objects. I'm not seeing a huge amount of value in distinguishing between Policy and Query objects in this case though -- now there are two places to look for complex read methods and it seems a bit fuzzy to me which methods belong where, especially considering that some of the Policy objects also contain complex queries. 

Should Policy objects and Query objects with the same name be merged into a single Policy object? If not, why not?

I don't mean to assert my opinion super strongly here, but having two places to put similar things can lend to duplication and increased complexity so it seems worth considering.

Reply:
here is one example of a blog post showing policy objects which contain some query-object-like methods: https://blog.eq8.eu/article/policy-object.html (grep for "Getting complex")

Reply:
My thinking is that Query objects are used when you want to retrieve model instances, and Policy objects are used when you want to get some meta-information about something. Their implementations may both involve database queries, but they are differentiated by their purpose rather than by their methods.

Reply:
that's a pretty clear rule of thumb -- let's keep them separate. I think this is reasonably well captured in the READMEs which I forgot to look at when I reviewed this PR.